fleetctl/fleetctl.go

```diff
@@ -130,6 +133,9 @@ func init() {
 	globalFlagset.StringVar(&globalFlags.CertFile, "cert-file", "", "Location of TLS cert file used to secure communication with the fleet API or etcd")
 	globalFlagset.StringVar(&globalFlags.CAFile, "ca-file", "", "Location of TLS CA file used to secure communication with the fleet API or etcd")
 
+	globalFlagset.StringVar(&globalFlags.EtcdUsername, "etcd-username", "", "Username used to authenticate with etcd")
+	globalFlagset.StringVar(&globalFlags.EtcdPassword, "etcd-password", "", "Password used to authenticate with etcd")
+
```

Comment:
hmm do we have any other config variables that can be set from the config file but not CLI?